.github/workflows/check.yaml

```
@@ -52,3 +52,6 @@ jobs:
 
       - name: Check typings
         run: npm run check:types
+
+      - name: Update supported Bee action
+        uses: ethersphere/update-supported-bee-action@v1.0.0
```

```suggestion
        uses: ethersphere/update-supported-bee-action@v1
        if: github.ref == 'refs/heads/master'
```

I would suggest running it only when updates are merged into master as they should be final, because you can for example use github install of bee-js for some "future preparation works" which could trigger the update even though the bee-js is not released or ready.

Also I would suggest to use `v1` tag to allow future non-breaking updates.

---

good feedback, thanks